tutorials/image-curation/image-curation.ipynb

```diff
@@ -681,6 +681,7 @@
     "    id_column_type=\"str\",\n",
     "    embedding_col=\"image_embedding\",\n",
     "    which_to_keep=\"hard\",\n",
+    "    batched_cosine_similarity=1024,\n",
```

Comment:
Has this been tested?

Reply:
Nope, do we always manually run these notebooks for such PRs? That'll be a time sink but I'm okay to do it that's the practice

Reply:
If it is expected to produce the same results as before, it is okay with me. Sometimes I leave notebooks unchanged (or add changes to have it keep the previous default) if the output is expected to change, so that the user won't be confused when their cell outputs are different than the ones on GitHub.

Reply:
But it sounds like that isn't the case here?

Reply:
It'll be the same result as before, and the notebook output should also not change